skimage/transform/_warps.py

```diff
@@ -209,6 +211,7 @@ def resize(
     return out
 
 
+@dispatchable
 @channel_as_last_axis()
 def rescale(
```

Comment:
In which order should the decorators be applied? dispatch first, then `channel_as_last_axis` or the other way around?

Reply:
I think almost always the first. This one does things to NumPy arrays, so it seems not guaranteed that it works with any other input someone might want to dispatch for.
An exception may be a decorator that just renames/deprecates some kwargs (i.e. clearly doesn't work with anything type specific, but is important for API transition).

Reply:
See https://github.com/scikit-image/scikit-image/pull/7723#discussion_r1986119270.